node/src/reactor/validator/memory_metrics.rs

```diff
@@ -86,9 +86,9 @@ impl MemoryMetrics {
         )?;
         let mem_block_proposer =
             IntGauge::new("mem_block_proposer", "block_proposer memory usage in bytes")?;
-        let mem_block_payload_validator = IntGauge::new(
-            "mem_block_payload_validator",
-            "block_payload_validator memory usage in bytes",
+        let mem_block_validator = IntGauge::new(
+            "mem_block_validator",
```

Comment:
CC @sacherjj @TomVasile looks like the metric `mem_block_payload_validator` is being changed to `mem_block_validator` here... not sure we are surfacing this in our dashboard.